<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-87BMGUIK" failures="0" tests="1" name="Default test" time="0.086" errors="0" timestamp="2022-02-24T16:07:26 IST">
  <testcase classname="assertion.readExcel" name="@BeforeTest getExcel" time="0.035">
    <failure type="java.io.FileNotFoundException" message="C:\Users\NamrathaChakravarthi\eclipse-workspace\SeleniumAssignment&amp;quot;C:\Users\NamrathaChakravarthi\OneDrive\Documents\Test.xlsx&amp;quot; (The filename, directory name, or volume label syntax is incorrect)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\NamrathaChakravarthi\eclipse-workspace\SeleniumAssignment"C:\Users\NamrathaChakravarthi\OneDrive\Documents\Test.xlsx" (The filename, directory name, or volume label syntax is incorrect)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at assertion.readExcel.getExcel(readExcel.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:645)
at org.testng.TestRunner.beforeRun(TestRunner.java:634)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest getExcel -->
  <testcase classname="assertion.readExcel" name="@AfterTest teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest teardown -->
  <testcase classname="assertion.readExcel" name="testExcel" time="0.0">
    <skipped/>
  </testcase> <!-- testExcel -->
</testsuite> <!-- Default test -->
